application.py

```diff
@@ -30,7 +36,20 @@ def index():
 def register():
     if request.method == "POST":
         username = request.form['username']
+        email = request.form['email']
         password = request.form['pswd']
-        print(username, password)
```

Comment:
use logging module instead of print statements for debugging. the advantage is that you can turn on the debug statements when needed.